analyzeme/src/profiling_data.rs

```diff
@@ -649,6 +649,86 @@ mod tests {
             );
         }
 
+        // To generate this revision, a v9 revision of the rust toolchain was
+        // created, and "rustup toolchain link" was used to name it "bespoke".
+        // Then, the following commands were executed:
+        //
+        //   # Make a small test binary and profile it.
+        //   cargo new --bin testbinary
+        //   cargo +bespoke rustc --bin testbinary -- -Zself-profile
```

Comment:
That does not mention using `mmedit` to trim the file. I assume you did that though, right? Otherwise I'd assume the file would be bigger, even for a small test binary.

Reply:
I actually didn't! I can, if we want, but by virtue of just using a `cargo new` binary, it already looks smaller than the v7 and v8 gzip'd examples.

Reply:
That's fine to then 🙂